pkgs/by-name/ti/tigerbeetle/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, git
+, zig_0_11
+, testers
+, tigerbeetle
+, nix-update-script
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tigerbeetle";
+  version = "0.14.171";
+
+  src = fetchFromGitHub {
+    owner = "tigerbeetle";
+    repo = "tigerbeetle";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-llcAR1xWNhbG27ph757ivK1ReU0RYB7amiZzkFchGGY=";
+    leaveDotGit = true;
```

Comment:
```suggestion
```
`leaveDotGit` can not be used in derivations as the contents under .git can change making the hash invalid, which it now is.

Reply:
Thank you for the review, that makes total sense now that I think about it. tigerbeetle's build process wants to figure out a git commit to include in the binary purely for debugging purposes, and `leaveDotGit` was my attempt to play nice with it.
I'm now forcing the commit to be a string of zeroes, since I don't really want to introduce too much maintenance overhead with a custom updater script.